movieApp.js

```diff
@@ -0,0 +1,170 @@
+document.addEventListener("DOMContentLoaded",  async ()=>{
+    await loadFilmTitles();
+    // await getFilmInfo()
+    await userReview();
+})
+
+const filmsAPI = 'https://ghibliapi.herokuapp.com/films';
+
+//function to get all of the film titles
+const loadFilmTitles =  async () => {
+
+// get the select element on the web page and let its initial value be 0 because there is no movie input.
+let dropdown = document.getElementById('movie-titles');
+dropdown.length = 1; 
+
+// the first dropdown option is blank because no movies have been populated.
+let defaultOption = document.createElement('option');
+defaultOption.text = ' ';
+
+// adding the blank option to the list
+dropdown.add(defaultOption);
+dropdown.selectedIndex = 0;
+
+    await axios
+        .get(filmsAPI)
+        .then((response)=>{
+            // console.log(response)
+            let films = response.data;
+            // console.log(films)
+            let option;
+            for (let i = 0; i < films.length; i++ ){
+                //creating the drop down list options 
+                option = document.createElement('option');
+                option.text = films[i].title;
+                option.value = ' ';
+                // displayFilmInfo(film);
+                dropdown.add(option)
+                // console.log(option)
+                film = films[i]
+                console.log(film)
+                // if(!dropdown.selectedIndex === 0){
+                //     console.log(dropdown.selecetedIndex) 
+                //  }
+            }
+          
+                
+
+
+                defaultOption.onchange = displayFilmInfo(option);
```

Comment:
You might want to put this code inside the for loop if your intention was to add onchange attribute to all option tags created from the loop.